index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```

Comment:
Should be in Spanish HTML
```suggestion
</mesa>
```

```suggestion
</mesa>
```